pkg/pkgmgr/dpkg.go

```diff
@@ -319,29 +319,29 @@ func (dm *dpkgManager) installUpdates(ctx context.Context, updates unversioned.U
 
 	var installCmd string
 	if updates != nil {
-		aptInstallTemplate := `sh -c "apt install --no-install-recommends -y %s && apt clean -y"`
+		aptGetInstallTemplate := `sh -c "apt-get install --no-install-recommends -y %s && apt-get clean -y"`
 		pkgStrings := []string{}
 		for _, u := range updates {
 			pkgStrings = append(pkgStrings, u.Name)
 		}
-		installCmd = fmt.Sprintf(aptInstallTemplate, strings.Join(pkgStrings, " "))
+		installCmd = fmt.Sprintf(aptGetInstallTemplate, strings.Join(pkgStrings, " "))
 	} else {
 		// if updates is not specified, update all packages
-		installCmd = `sh -c "output=$(apt upgrade -y && apt clean -y && apt autoremove 2>&1); if [ $? -ne 0 ]; then echo "$output" >>error_log.txt; fi"`
+		installCmd = `sh -c "output=$(apt-get upgrade -y && apt-get clean -y && apt-get autoremove 2>&1); if [ $? -ne 0 ]; then echo "$output" >>error_log.txt; fi"`
```

Comment:
it seems like this command is not downloading the packages, resulting in the error for the test cases with no report

Reply:
let me check

Reply:
@ashnamehrotra, could you check [it](https://github.com/project-copacetic/copacetic/actions/runs/11266328802/job/31329688791?pr=804#step:9:107) once, and [this](https://github.com/project-copacetic/copacetic/actions/runs/11266328802/job/31329688791?pr=804#step:9:263) too

Reply:
it looks like results manifest file is not found, so the packages are not found/downloaded correctly. There is also a log output `line 2: packages.txt: No such file or directory`. Would you be able to run the apt-get commands manually in a container to see if they are working as expected?